.azure/pipelines/quarantined-pr.yml

```diff
@@ -1,9 +1,46 @@
-# We want to run quarantined tests on master as well as on PRs
+#
+# See https://docs.microsoft.com/en-us/vsts/pipelines/yaml-schema for details on this file.
+#
+
+# Configure which branches trigger builds
+# We want to run quarantined tests on release/5.0 and master as well as on PRs
 trigger:
   batch: true
   branches:
     include:
     - master
+    - release/5.0
+  paths:
+    exclude:
+    - .github/*
+    - .vscode/*
+    # Exclude only a few specific Markdown files because some of them ship.
+    - docs/*
+    - CODE-OF-CONDUCT.md
+    - CONTRIBUTING.md
+    - LICENSE.TXT
+    - README.md
+    - SECURITY.md
+    # Do not ignore the THIRD-PARTY-NOTICES.TXT file because it ships.
+
+# Run PR validation on branches that include Helix tests
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - release/5.0
+    - master
+  paths:
+    exclude:
+    - .github/*
+    - .vscode/*
+    - '**/*.md'
+    - CODE-OF-CONDUCT.md
+    - CONTRIBUTING.md
+    - LICENSE.TXT
+    - README.md
+    - SECURITY.md
+    - THIRD-PARTY-NOTICES.TXT
```

Comment:
The settings in the release/5.0 branch include all target branches. We're probably fine there because doc-only updates during servicing are incredibly rare.